src/pages/HomeParticipant.vue

```diff
@@ -9,19 +9,42 @@
         <h2 v-if="adminInfo?.length == 1" class="p-float-label dropdown-container">
           {{ adminInfo.at(0).name }}
         </h2>
-        <div v-if="adminInfo?.length > 1" class="p-float-label dropdown-container">
+        <div
+          v-if="adminInfo?.length > 1"
+          class="flex flex-row justify-center align-items-center p-float-label dropdown-container gap-4"
+        >
           <PvDropdown
             v-model="selectedAdmin"
             :options="adminInfo ?? []"
             option-label="name"
             input-id="dd-assignment"
             data-cy="dropdown-select-administration"
           />
+          <PvInputSwitch v-model="showOptionalAssessments" data-cy="switch-show-optional-assessments" />
+          <p>Show Optional Assessments</p>
           <label for="dd-assignment">{{ $t('homeParticipant.selectAssignment') }}</label>
```

Comment:
Suggestions:
- Reorder the labels and primevue components so that the labels are next to the components.
- Change `p` tag to a `label`
- Use `input-id` to associate the underlying input element with the label
- Use `$t` to support internationalization later on
```suggestion
          <label for="dd-assignment">{{ $t('homeParticipant.selectAssignment') }}</label>
          <PvInputSwitch v-model="showOptionalAssessments" input-id="switch-optional" data-cy="switch-show-optional-assessments" />
          <label for="switch-optional">{{ $t('homeParticipant.showOptionalAssessments') }}</label>
```